zebra-network/Cargo.toml

```diff
@@ -38,6 +38,7 @@ zebra-chain = { path = "../zebra-chain" }
 [dev-dependencies]
 proptest = "0.10"
 proptest-derive = "0.3"
+tokio = { version = "0.3.6", features = ["test-util"] }
```

Comment:
Do we need to repeat the non-dev features here, or does cargo just assume we want them all as well?
(I mean obviously whatever we're doing is working, so 🤷 .)

Reply:
I know that cargo does feature supersets to improve compilation time. I just didn't realise they worked this well.